IAMOTZ/OOP-Warrior
# set up my project to work with es6

#### What does this PR do?
It makes the project to be able to work with es6 javacscript
#### How should this be manually tested?
After cloning the repo, 
- try manipulate the main.js to see meaningful changes
- open the command prompt 
- cd into the project directory
- RUN `npm run webpack`
- open the build/main.build.js to see the es5 version
